modules/checklist/steps-manager.php

```diff
@@ -159,6 +163,6 @@ private function get_step_instance( string $step_id ) : ?Step_Base {
 	 * @return array
 	 */
 	private static function get_step_ids() : array {
-		return [ Create_Pages::STEP_ID ];
+		return [ Create_Pages::STEP_ID, Setup_Header::STEP_ID ];
```

Comment:
I am not sure how this method is supposed to work. How does 3rd party steps get in here?

Reply:
I'll make a static property, and when the filter is implemented we'll merge or do some kind of operation, wdyt?

Reply:
Independent task for that